### PR TITLE
[Excel] (requirement sets) Update Preview and Online API lists

### PR DESCRIPTION
**Pull request by alison-mk**
Pulling latest from DefinitelyTyped.

### CODE REVIEW

File: docs/reference/requirement-sets/excel-api-online-requirement-set.md [hunk 1]
```diff
@@ -37,7 +37,23 @@ Once the API is in a cross-platform requirement set, you should remove or edit t
 
 | Class | Fields | Description |
 |:---|:---|:---|
-|[Range](/javascript/api/excel/excel.range)|[getMergedAreas()](/javascript/api/excel/excel.range#getmergedareas--)|Returns a RangeAreas object that represents the merged areas in this range.|
+|[NamedSheetView](/javascript/api/excel/excel.namedsheetview)|[activate()](/javascript/api/excel/excel.namedsheetview#activate--)|Activates this sheet view.|
```
Comment (AlexJerabek) [inline docs/reference/requirement-sets/excel-api-online-requirement-set.md:40]: @ElizabethSamuel-MSFT Where are we on the link anchor issue with OPS?

File: docs/reference/requirement-sets/excel-preview-apis.md [hunk 2]
```diff
@@ -19,7 +19,6 @@ The first table provides a concise summary of the APIs, while the subsequent tab
 | Document tasks | Turn comments into tasks assigned to users. | [DocumentTask](/javascript/api/excel/excel.documenttask) |
 | Formula changed events | Track changes to formulas, including the source and type of event that caused a change. | [Worksheet.onFormulaChanged](/javascript/api/excel/excel.worksheet#onFormulaChanged)|
 | Linked data types | Adds support for data types connected to Excel from external sources. | [LinkedDataType](/javascript/api/excel/excel.linkeddatatype)|
-| Named sheet views | Gives programmatic control of per-user worksheet views. | [NamedSheetView](/javascript/api/excel/excel.namedsheetview) |
```
Comment (AlexJerabek) [inline docs/reference/requirement-sets/excel-preview-apis.md:22]: Can we move this to the Online Only requirement set page? Let's add a similar table to that article (even if this is the only item it contains).

**Review by AlexJerabek — APPROVED** [review]
Approving, with one change request.

File: docs/reference/requirement-sets/excel-api-online-requirement-set.md (hunk 1 above)
Comment (ElizabethSamuel-MSFT) [reply docs/reference/requirement-sets/excel-api-online-requirement-set.md:48]: Current estimate is this quarter

**Review by AlexJerabek — APPROVED** [review]
Looks good. I like the placement of the new table. It actually makes me wonder in the preview API table is in the right place, with respect to the note.

**Pull request merged**